src/Raven.Server/Documents/PeriodicExport/PeriodicBackupStore.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Raven.Client.Server.PeriodicExport;
+using Raven.Server.ServerWide.Context;
+using Sparrow;
+using Sparrow.Json;
+using Voron;
+
+namespace Raven.Server.Documents.PeriodicExport
+{
+    public unsafe class PeriodicBackupStore
+    {
+        private static readonly Slice PeriodicExportStatusSlice;
+        
+
+        static PeriodicBackupStore()
+        {
+            Slice.From(StorageEnvironment.LabelsContext, "PeriodicExportStatus", ByteStringType.Immutable, out PeriodicExportStatusSlice);
+        }
+
+        public BlittableJsonReaderObject GetDatabasePeriodicBackupStatus(DocumentsOperationContext context)
+        {
+            var tree = context.Transaction.InnerTransaction.ReadTree(PeriodicExportStatusSlice);
```

Comment:
I think we want to use Configuration env here, otherwise we'll be blocking the documents.
In general, I like that we are splitting this up.